source/conf.py

```diff
@@ -355,6 +355,9 @@ def setup(app):
     r'https://arxiv.org/.*', # Sometimes returns 403 on Travis (rate limit?)
     r'https://www.researchgate.net/.*', # Started returning 403 on Travis
     r'https://symbolblog.com/.*', # Returning 403
+    r'https://docs.github.com/.*', # GitHub is now returning 403
+    r'https://help.github.com/.*', # GitHub is now returning 403
+    r'https://github.com/fboucquez/symbol-bootstrap#command-topics', # Returning Anchor 'command-topics' not found
```

Comment:
I'd really prefer that we do not overpopulate this list... this is weakening the link check :/
I think the right anchor in this case is:
https://github.com/fboucquez/symbol-bootstrap#user-content-command-topics

Also, all these 403 might be related to throttling? The link checker has a mechanism to avoid making too many consecutive requests to the same server, but maybe changing the CI has altered that?

Reply:
Thanks Xavi, I think I found the issue with 403 error.  If I add with user_agent it seems to fix the problem

https://github.com/sphinx-doc/sphinx/issues/7369#issuecomment-605009428

Reply:
The updated link for bootstrap also fix the issue

Reply:
Awesome!